edx_django_utils/cache/tests/test_utils.py

```diff
@@ -168,6 +168,35 @@ def test_get_cached_response_django_cache_hit(self, mock_cache_get):
         cached_response = self.request_cache.get_cached_response(TEST_KEY)
         self.assertTrue(cached_response.is_found, 'Django cache hit should cache value in request cache.')
 
+    @mock.patch('edx_django_utils.cache.utils.waffle.switch_is_active')
```

Comment:
See https://waffle.readthedocs.io/en/stable/testing/automated.html for `override_switch` decorator for testing.

Reply:
Thanks for mentioning it.
The reason I didn't use it at first because our `TestTieredCache` is inherited from `unittest.Testcae` which doesn't allow the database access and we get the following error when we try to check the `toggle` status.

```
*** RuntimeError: Database access not allowed, use the "django_db" mark, or the "db" or "transactional_db" fixtures to enable it.
```

BUT I've changed the parent class. It's working fine now.